openslides/utils/models.py

```diff
@@ -22,7 +27,7 @@ def formfield(self, **kwargs: Any) -> Any:
         return super(MinMaxIntegerField, self).formfield(**defaults)
 
 
-class RESTModelMixin:
+class RESTModelMixin():
```

Comment:
Why do we need the brackets?

Reply:
Typo. 

I tried to make the RESTModelMixin class(!) a Cachable be adding Cachable as a metaclass. This did not work but I did forget to remove the brackets.